### DISCUSSION TITLE
reDeployment issue

### DISCUSSION THREAD

**Discussion by martinbauwens** [vercel/community#3989] [Help] [answered]
### Summary

framework: nextjs

Dear all,

I am a newbie on Vercel.

I am getting an error, that I am unable to eliminate, when trying to redeploy (see below picture):

<img width="485" alt="Screenshot 2023-09-09 at 14 10 05" src="https://github.com/vercel/community/assets/143646690/5da17a77-3239-4faf-980b-1391e83d7ece">

By reverting to the initial code and committing it, I removed the error in the code which was causing the error, but I am still unable to redeploy.

Can anybody assist?

I did a search, but did not find a solution.

Thank you in anticipation.

Regards,

Martin

### Steps to Reproduce

See above message

**Comment by swarnava** [comment]
HI Martin,

If you already committed your fixed code, it should automatically generate a new production build. Are you seeing it otherwise? Can you share us a screencast?

**Comment by martinbauwens** [comment]
Hi Swarnava,

Thank you for your reply.

I am still seeing the below error, despite the succesful commitment made a few hours ago:

<img width="1232" alt="Screenshot 2023-09-09 at 18 17 17" src="https://github.com/vercel/community/assets/143646690/9dbc2227-7615-4ab3-9633-406b9bd8b35f">

Regards,

Martin

**Reply by swarnava** [reply]
That looks like a Build error not Vercel error. You can run the build command locally and reproduce this.

**Reply by martinbauwens** [reply]
"...and reproduce this" - what does this mean (why would I reproduce an error)?

**Reply by swarnava** [reply]
If you run the build command (`npm run build && npm start` or similar depending upon the framework you are using) in your local machine, you should receive the similar error which indicates the issue with the code.

**Reply by martinbauwens** [reply]
PS. I never said it was a Vercel error. I am looking for a solution not for someone to blame.

**Reply by martinbauwens** [reply]
The code that caused the issue has been removed. I reverted to the original code which had no error. The original code was committed without me receiving an error message from Vercel.

**Reply by martinbauwens** [reply]
Is this an indication that there is no longer a problem?

<img width="1127" alt="Screenshot 2023-09-09 at 19 11 51" src="https://github.com/vercel/community/assets/143646690/223f4fcf-d11c-4f16-9f6d-28d53eff1fd5">

**Reply by swarnava** [reply]
I can't see if it's your latest production deployment, but if that's the case, yes you have the latest commit running on Production then :)

**Reply by martinbauwens** [reply, marked as answer]
Yes, this is the latest. Thank you.
